### PR TITLE
Update to use  Halogen RC5 and PS 0.13

### PR DESCRIPTION
**Pull request by Unisay**

### CODE REVIEW

File: spago.dhall [hunk 1]
```diff
@@ -1,5 +1,9 @@
-{ name = "purescript-halogen-realworld"
-, packages = ./packages.dhall
+{ sources =
+    [ "src/**/*.purs", "test/**/*.purs" ]
+, name =
+    "purescript-halogen-realworld"
+, packages =
+    ./packages.dhall
```
Comment (Unisay) [inline spago.dhall:6]: Result of `dhall format --inplace spago.dhall`

File: src/Page/Home.purs [hunk 2]
```diff
@@ -242,7 +242,7 @@ component = H.mkComponent
         [ HH.i
           [ css "ion-pound" ]
           []
-        , HH.text $ "\160" <> tag
+        , HH.text $ "\x160" <> tag
```
Comment (Unisay) [inline src/Page/Home.purs]: I am not 100% sure about this one

Comment (thomashoneyman) [reply src/Page/Home.purs]: I _think_ that's just a hash -- so we could stick "#" in there directly. I never remember the codes myself.

File: packages.dhall [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
+      https://raw.githubusercontent.com/purescript/package-sets/psc-0.13.2-20190725/src/packages.dhall sha256:60cc03d2c3a99a0e5eeebb16a22aac219fa76fe6a1686e8c2bd7a11872527ea3
+
 
 let overrides =
       { halogen =
-          upstream.halogen // { version = "v5.0.0-rc.4" }
+          upstream.halogen // { version = "v5.0.0-rc.5" }
       , halogen-vdom =
           upstream.halogen-vdom // { version = "v6.1.0" }
       , remotedata =
```
Comment (thomashoneyman) [inline packages.dhall]: I believe `remotedata` at this point is in the regular set and doesn't need to be overriden

**Comment by thomashoneyman** [comment]
@Unisay If you update the hash and check on whether `remotedata` can be removed from the overrides then I'd love to merge this in. Thanks!

File: packages.dhall (hunk 3 above)
Comment (Unisay) [reply packages.dhall]: Cool, I'll check if I can remove it

**Comment by Unisay** [comment]
@thomashoneyman will do shortly

File: src/Page/Home.purs (hunk 2 above)
Comment (Unisay) [reply src/Page/Home.purs]: Done

**Review by thomashoneyman — APPROVED** [review]

**Comment by thomashoneyman** [comment]
Thanks!

**Pull request merged**